src/main/webapp/app/shared/statistics-graph/statistics.service.ts

```diff
@@ -62,4 +65,11 @@ export class StatisticsService {
         stats.absoluteAveragePoints = round((stats.averageScoreOfExercise * stats.maxPointsOfExercise) / 100, 1);
         return stats;
     }
+
+    private static convertDatesFromServer(dto: CourseManagementStatisticsDTO): CourseManagementStatisticsDTO {
+        dto.averageScoresOfExercises.forEach((averageScores) => {
+            averageScores.releaseDate = averageScores.releaseDate !== null ? moment(averageScores.releaseDate) : undefined;
```

Comment:
```suggestion
            averageScores.releaseDate = averageScores.releaseDate != undefined ? moment(averageScores.releaseDate) : undefined;
```

Reply:
Could be an improvement to cheeck for undefined as well. However, since these values are straight from the server, releaseDate should still be null and not undefined, or not?